synapse/lib/stormtypes.py

```diff
@@ -7807,6 +7812,10 @@ async def _methViewSet(self, name, valu):
             valu = await toprim(valu)
 
         elif name == 'nomerge':
+            name = 'protected'
```

Comment:
Should we warn about the deprecation here? It seems like it could get spammy pretty quickly.